custom-words.txt

```diff
@@ -1921,6 +1921,8 @@ serverless
 serverlevel
 servicebus
 serviceconfiguration
+serviceconfigurations
+Serviceconfigurations
```

Comment:
We shouldn't need these in custom-words.txt if we always PascalCase and CamelCase them. I wonder why we don't.